cpg_infra/billing_aggregator/aggregate/requirements.txt

```diff
@@ -5,8 +5,9 @@ metamist
 pyyaml
 python-rapidjson
 google-api-python-client==2.79.0
-google-cloud-bigquery==3.6.0
+google-cloud-bigquery==3.11.4
 google-cloud-secret-manager==2.15.1
 google-cloud-logging==3.5.0
-pandas==1.5.3
-pandas-gbq==0.19.1
+numpy==1.23.2
```

Comment:
Is numpy just pinned, or was it introduced?

Reply:
It is pinned, I was having issue with pip install during cloud function setup with numpy 2.0.0 which is now default. This version was suggested on stackoverflow, I just can't find the reference atm.